GETTING_STARTED.md

```diff
@@ -0,0 +1,35 @@
+## Setup NServiceBus.RabbitMQ
+
+- Install virtualization software like VirtualBox
+- Create Win10 VM
```

Comment:
Why the need to use a VM? I've done plenty of development against RabbitMQ and I've never had to set up a VM. I just install it locally.

I can see that this may be a valuable optional way to do it, but if we imply that you *have* to use a VM it feels like quite a barrier to entry.